source/config/__init__.py

```diff
@@ -315,6 +315,16 @@ def setStartOnLogonScreen(enable):
 		if execElevated(SLAVE_FILENAME, (u"config_setStartOnLogonScreen", u"%d" % enable), wait=True) != 0:
 			raise RuntimeError("Slave failed to set startOnLogonScreen")
 
+
+def openConfigurationDirectory():

```

Comment:
Strictly spoken, the system config directory is a configuration directory as well.

```suggestion
def openUserConfigDirectory():
```

Reply:
Strictly spoken you can set serviceDebug to False in the registry and use both menu item and keyboard command to open systemConfig on secure screens.

Reply:
It is also worth discussing if directory containing configuration in case of portable copies can be called user configuration directory.

Reply:
True, but in the portable copy, the directory itself is called userConfig

Reply:
I haven't realized that it is 'userconfig' as in the directory name and not as directory containing user configuration files. In this case it is even more incorrect, as it is possible to specify alternative config directory via command line flag, and obviously it can be named anything.

Reply:
I think I would prefer this named `openUserConfigDirectory` not because it relates to the name of the folder, but because it relates to the name of the contents. It is the configuration that belongs to the active user. In some situations the active user may be `system`.